examples/wakaama/lwm2m_cli.c

```diff
@@ -21,22 +21,68 @@
 #include "lwm2m_client.h"
 #include "lwm2m_client_objects.h"
 #include "lwm2m_platform.h"
+#include "objects/security.h"
+#include "objects/device.h"
+
+#include "net/credman.h"
 
 #define OBJ_COUNT (3)
 
 uint8_t connected = 0;
 lwm2m_object_t *obj_list[OBJ_COUNT];
 lwm2m_client_data_t client_data;
 
+#ifndef CONFIG_PSK_ID
+#define CONFIG_PSK_ID "Client_Identity"
```

Comment:
Does this macro _has_ to have this name by wakaama or can we give it a less conflicting name? E.g.

```suggestion
#define CONFIG_WAKAAMA_PSK_ID "Client_Identity"
```

Reply:
This is application configuration, it can be changed to something else

Reply:
I know, but let's try to prevent conflicts wherever possible :-)

Reply:
done